mlir/python/mlir/presburger.py

```diff
@@ -0,0 +1,12 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Simply a wrapper around the extension module of the same name.
+from ._mlir_libs import _mlirPresburger as _presburger
+
+__all__ = ["IntegerRelation"]
+
+
+class IntegerRelation(_presburger.IntegerRelation):
+    pass
```

Comment:
nit: i don't think this is useful - everywhere else we just do
```suggestion
from ._mlir_libs._mlirPresburger import *

```

Reply:
This is also fine. I have made this change.